observability/micrometer/src/test/java/org/wildfly/extension/micrometer/SubsystemParsingTestCase.java

```diff
@@ -24,11 +23,13 @@ public static Iterable<MicrometerSubsystemSchema> parameters() {
     }
 
     public SubsystemParsingTestCase(MicrometerSubsystemSchema schema) {
-        super(MicrometerConfigurationConstants.NAME, new MicrometerExtension(), schema, MicrometerSubsystemSchema.CURRENT);
+        super(MicrometerConfigurationConstants.NAME, new MicrometerExtension(), schema,
+            MicrometerSubsystemSchema.VERSION_1_1_COMMUNITY);
     }
 
     @Override
-    protected String getSubsystemXmlPathPattern() {
-        return "%s_%d_%d.xml";
+    public void testSubsystem() throws Exception {
+        System.out.printf("Testing " + getSubsystemSchema());
+        super.testSubsystem();
```

Comment:
I'll clean this up after I see how CI does with these changes over all. I've not had good luck on CI with force pushes in rapid succession. :)